tests/filtertransformers/test_mongo.py

```diff
@@ -627,12 +697,7 @@ def test_list_properties(self):
             "$and": [
                 {"elements": {"$in": ["H"]}},
                 {"elements": {"$all": ["H", "He", "Ga", "Ta"]}},
-                {
-                    "elements": {
-                        "$all": ["H", "He", "Ga", "Ta"],
-                        "$size": 4,
-                    }
-                },
+                {"elements": {"$all": ["H", "He", "Ga", "Ta"], "$size": 4}},
```

Comment:
(and here with white space)

Reply:
I'll try to put the original white space back.

Reply:
Black again places everything on one line after I tried to change it back to the original formatting.